packages/overlay/src/integrations/sentry/components/events/error/Frame.tsx

```diff
@@ -51,9 +51,10 @@ function FileActions({ frame }: { frame: EventFrame }) {
     return null;
   }
   const resolvedFilename = resolveFilename(frame.filename);
+  const canOpenFileInEditor = !frame.filename?.includes(':');
```

Comment:
Ah, this removal was intentional. We should not need this anymore. Did you find an edge case?

Reply:
Not edge case, but i thought we are giving the ability to open specific files to get to the error.
Not sure if opening other build files makes much sense.
but we can do this.
reverting it